src/test/java/seedu/task/testutil/SimpleTaskBuilder.java

```diff
@@ -77,7 +87,11 @@ public SimpleTaskBuilder withAlertWindow(String alertWindow) {
     }
 
     public Task build() {
-        return new SimpleTask(name, description, tags);
+        return new SimpleTask(name, description, tags, effort);
+    }
+
+    public Task buildDefault() {
+        return new SimpleTask(name, description, tags, null);
     }
```

Comment:
null value being passed into the object seems to be dangerous. How about ```new effort()```?

Reply:
Good job noticing that! This is actually another method I created to test if SimpleTask automatically uses a default effort value when the user does not key it in. The regular build() method used to build other SimpleTask stubs is still present!